hw/ip_templates/clkmgr/dv/env/clkmgr_env_cfg.sv.tpl

```diff
@@ -16,10 +16,14 @@ class clkmgr_env_cfg extends cip_base_env_cfg #(
   virtual clkmgr_csrs_if clkmgr_csrs_vif;
   virtual clk_rst_if     main_clk_rst_vif;
   virtual clk_rst_if     io_clk_rst_vif;
+  virtual clk_rst_if     io_div2_clk_rst_vif;
```

Comment:
Do the list of VIFs here need to be generated? There are main clocks and derived clocks here.

Reply:
Yes, it will be generated. I fixed a couple other places in clkmgr dv.

Reply:
Done, also removed usb from ast_clks_t.